index.html

```diff
@@ -10,7 +10,28 @@
 </head>
 
 <body>
-    <h1>Hello World!</h1>
+    <!-- home page header -->
+    <main>
+        <header>
+            <div class="mobile_icon">
+                <p>9:41</p> 
+                <img src="images/bar.png" alt="" height="15">
+            </div>
+            <div class="static_bar">
+                <p class="name">My Logo</p>
+                <!-- Navigation links -->
+                <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-menu-2 "
+                 width="44" height="25" viewBox="0 0 24 24" stroke-width="1.5" stroke="#6070ff" 
+                 fill="none" stroke-linecap="round" stroke-linejoin="round">
+                    <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
+                    <line x1="4" y1="6" x2="20" y2="6" />
+                    <line x1="4" y1="12" x2="20" y2="12" />
+                    <line x1="4" y1="18" x2="20" y2="18" />
+                  </svg>
+            </div>  
+        </header>  
```

Comment:
Below the `header`, there should be a new section `the headline` that is wrapped inside a `section` tag. You are missing the entire headline section as per project requirements.

Your headline:

![image](https://user-images.githubusercontent.com/38088818/185604274-0d990fb7-8376-49bc-8973-1f6219fe8814.png)

Expected Headline:

![image](https://user-images.githubusercontent.com/38088818/185604430-874ba521-3363-4ded-b4d3-6d9bf2d3db16.png)

Key points to consider when creating the headline section.

- The heading with the title "I'm Mariane" must be an `h1` tag.
- The paragraph with the description must use a `p` tag.
- The social icons must use the `ul` and `li` tags and each social link must contain the `a` tag since they are links that should redirect users to different social media websites.